alert-database/src/main/java/com/synopsys/integration/alert/database/BaseEntity.java

```diff
@@ -30,6 +30,9 @@
 
 @MappedSuperclass
 public abstract class BaseEntity extends Stringable implements Serializable {
+    public static final int DEFAULT_ALLOCATION_SIZE = 50;
+    private static final long serialVersionUID = 5939107067779376362L;
```

Comment:
If I recall correctly, we don't want to declare this because we are not maintaining multiple versions of the same java objects. You should be able to turn off the warning and generation of this from your Intellij settings.

Reply:
I will disable it and fix the files I changed that added it.